docs/policygenerator-reference.yaml

```diff
@@ -200,4 +212,14 @@ policySets:
       placementPath: ""
       # To reuse an existing placement rule manifest, specify the path here relative to the
       # kustomization.yaml file. (See clusterSelectors to generate a new PlacementRule instead.)
-      placementRulePath: ""
\ No newline at end of file
+      placementRulePath: ""
+      # Advanced configuration to use a placement that already exists in the cluster in the same
+      # namespace as the policy will be generated for.  It is the reponsiblity of the administrator to 
+      # ensure the placement exists.  Use of this setting will prevent a Placement from
+      # being generated.  The Binding will still be created.
+      placementName: ""
+          # Advanced configuration to use a placement rule that already exists in the cluster in the same
+      # namespace as the policy will be generated for.  It is the reponsiblity of the administrator to 
+      # ensure the placement rule exists.  Use of this setting will prevent a PlacementRule from
+      # being generated.  The Binding will still be created.
+      placementRuleName: ""
```

Comment:
This can be removed once this PR is reviewed/merged:
- https://github.com/stolostron/policy-generator-plugin/pull/52

Reply:
Thank you, I will merge that change in once it is reviewed/merged